packages/punbb.php

```diff
@@ -127,12 +127,29 @@ protected function forumExport($ex) {
 
         // Signatures.
         $ex->exportTable('UserMeta', "
-         SELECT
-         id,
-         'Plugin.Signatures.Sig' AS Name,
-         signature
-      FROM :_users u
-      WHERE u.signature IS NOT NULL", array('id ' => 'UserID', 'signature' => 'Value'));
+        select
+           u.id as UserID,
+           'Plugin.Signatures.Format' AS Name,
+           'BBCode' as Value
+        from 
+           :_users u
+        where 
+            u.signature is not null 
+        and 
+            u.signature != ''
+        
+        union
+        
+        select
+            u.id as UserID,
+            'Plugin.Signatures.Sig' AS Name,
+            signature as Value
+        from 
+            :_users u
+        where 
+            u.signature is not null 
+        and 
+            u.signature !='' ", array('id ' => 'UserID', 'signature' => 'Value'));
```

Comment:
Because you are using aliases in your query, you don't need to pass this 3rd parameter anymore.